components/alert/check-props.js

```diff
@@ -3,51 +3,55 @@
 /* eslint-disable import/no-mutable-exports */
 
 import sunsetProperty from '../../utilities/warning/sunset-property';
+import getComponentDocFn from '../../utilities/get-component-doc';
```

Comment:
Retrieves utility function that is used to print comment pointing to URL of prop or component (depending on what the user passes in to the function).